backend/config/accounts.go

```diff
@@ -43,3 +43,15 @@ func newDefaultAccountsonfig() AccountsConfig {
 		Accounts: []Account{},
 	}
 }
+
+// ByCode returns the account with the given code, or nil if no such account exists.
+// A reference is returned, so the account can be modified by the caller.
+func (cfg AccountsConfig) ByCode(code string) *Account {
```

Comment:
Will there be other `ByXxx` methods? If not, I think a better name might have been `Find` or `LookUp` or just `Account`.

Reply:
done